src/clr-angular/popover/signpost/signpost-content.ts

```diff
@@ -67,10 +67,12 @@ export class ClrSignpostContent extends AbstractPopover {
   /**********
    *
    * @description
-   * Close function that uses the signpost instance to toggle the state of the content popover.
+   * Close function that uses the IfOpenService instance to toggle the state of the content popover.
    *
    */
-  close() {
+  close(event: MouseEvent) {
+    event.preventDefault();
```

Comment:
Depending on how the consumer is using the signpost I don't think we can use `preventDefault` here. I remember it has always been an issue in the past when proposed as a solution.

Reply:
As a library/framework, we try to avoid changing the default behavior of the browsers. The reason is that our users expect those default behaviors and their app functionalities could very well depend on them. So we try to avoid disabling or changing them. Unfortunately, both `event. preventDefault` and `event.stopPropagation` changes the default behaviors. If necessary, we could use only one of them, maybe? Based on that, I see that we might not need `event.preventDefault` here becuase the error was due to event bubbling.

Reply:
It looks like `event.preventDefault` is needed to fix the issue, removing `event.stopPropagation` works but I'll need to adjust the tests a bit. I'm going to keep looking to see if there is any other ways I can get it to work without using either.

Reply:
I am not really sure this is actually a Clarity issue. Any button in a label will be exchanging events:
https://stackblitz.com/edit/button-in-a-label-issue
On the other hand, we definitely do not want this behavior, nor it is good to blame it on the user.
Preventing default may be justified here. Or we might want to reconsider if we need our signpost to be implemented as button in the 1st place.